test/components/user_list_item_test.js

```diff
@@ -45,6 +59,57 @@ describe("UserListItem", () => {
     })
   })
 
+  describe("passed a non-facilitator as a facilitator", () => {
+    const user = { ...defaultUserAttrs, is_facilitator: true }
+    const user2 = { ...secondaryrUserAttrs, is_facilitator: false }
+
+    it("renders the magic icon", () => {
+      const wrapper = shallow(
+        <UserListItem {...defaultProps} user={user2} currentUser={user} />
+      )
+      expect(wrapper.find("i.magic.icon")).to.have.length(1)
+    })
+
+    it("renders facilitator button", () => {
```

Comment:
__Note to self:__ check if these are separate buttons in implementation

Reply:
the button contains the icon inside of it

That logic is there so that the magic icon doesn't render for your _own_ avatar. Not that it's particularly harmful to pass facilitstorship to yourself, but in the moment it made more sense to put a bit of logic to prevent it